### PR TITLE
Update hints.json

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
None

#### Purpose of change

 Fixes strange NPC "hints" 

#### Describe the solution

 Just some simple edits.

#### Describe alternatives you've considered

 N/A

### CODE REVIEW

File: data/json/npcs/hints.json [hunk 1]
```diff
@@ -134,8 +134,8 @@
       "When the whole town is one big supermarket a shopping cart becomes your best friend.",
       "A friend is a second mouth to fill, but when it comes to work four hands are always better than two.",
       "I was against drugs until I was almost killed by a zombie.  I was hauling my sorry ass away from the horde, with nothing more but some white powder I got from that zombie.  Saved me that time.",
-      "Not sure if Mike is sane any more.  He was unfortunate enough to be driven in to a school one time.  This experience broke more than his ribs then.",
-      "A thought about explosives.  If you can still run and it did not went boom yet, run some more.  There is no such thing as excess space between you and a stick of dynamite.",
+      "A friend of mine ran into an elementary school, trying to hide from the hordes. I never saw him again.",
```
Comment (Maleclypse) [inline data/json/npcs/hints.json:137]: The original is warning about the morale hit from killing children zombies.

**Comment by Inglonias** [comment]
These hints have always felt odd to me. Most of the time you get them from the starting NPC, which is odd because the Cataclysm just happened, and these people already have this helpful advice.

**Comment by cincident** [comment]
> These hints have always felt odd to me. Most of the time you get them from the starting NPC, which is odd because the Cataclysm just happened, and these people already have this helpful advice.

I feel this way too. If they're in the game, though, then I'd like them to be sensible.

**Comment by kevingranade** [comment]
You pushed some extraneous commits adding a giant turtle, those need to be backed out.  Also it has a missing comma.

File: data/json/npcs/hints.json [hunk 2]
```diff
@@ -134,8 +134,8 @@
       "When the whole town is one big supermarket a shopping cart becomes your best friend.",
       "A friend is a second mouth to fill, but when it comes to work four hands are always better than two.",
       "I was against drugs until I was almost killed by a zombie.  I was hauling my sorry ass away from the horde, with nothing more but some white powder I got from that zombie.  Saved me that time.",
-      "Not sure if Mike is sane any more.  He was unfortunate enough to be driven in to a school one time.  This experience broke more than his ribs then.",
-      "A thought about explosives.  If you can still run and it did not went boom yet, run some more.  There is no such thing as excess space between you and a stick of dynamite.",
+      "I knew an elementary school teacher who had to fight his way out of school.  He had to kill a dozen crazed children.  I can't imagine what that does to a person.",
+      "A thought about explosives: If you can still run, and the explosive hasn't detonated yet, run some more.  No amount of caution is excessive when handling a grenade."
```
Comment (ZhilkinSerg) [inline data/json/npcs/hints.json:138]:
```suggestion
      "A thought about explosives: If you can still run, and the explosive hasn't detonated yet, run some more.  No amount of caution is excessive when handling a grenade.",
```

**Pull request merged**